style.css

```diff
@@ -160,3 +178,250 @@ nav a {
   color: #fff;
   text-decoration: none;
 }
+
+.card-block .card-pic {
+  position: relative;
+  height: 338px;
+  background:
+    linear-gradient(
+      180.45deg,
+      rgba(
+        38,
+        38,
+        38,
+        0
+      )
+      0.75%,
+      rgba(
+        38,
+        38,
+        38,
+        0.9
+      )
+      61.94%
+    ),
+    url(
+      "resources/aura.png"
+    );
+}
+
+.card-block .card-pic .card-info {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  height: 338px;
+  color: #fff;
+  padding: 5%;
+}
+
+.card-block .card-pic .card-info .card-title {
+  font-family: 'Crete Round', serif;
+  font-size: 32px;
+  margin-bottom: 10px;
+}
+
+.card-block .card-discription {
+  font-family: 'Inter', sans-serif;
+  margin-bottom: 10px;
+}
+
+.card-block .card-lang {
+  display: flex;
+}
+
+.card-block .card-lang > div {
+  padding: 10px;
+  background: rgba(255, 255, 255, 0.24);
+  margin-right: 10px;
+}
+
+.card-block .seemore a {
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  color: #fff;
+  font-weight: 700;
+  text-decoration: none;
+  display: block;
+  height: 48px;
+  width: 100%;
+  background: #ff6b00;
+  text-align: center;
+  line-height: 48px;
+}
+
+.works:last-child {
+  margin-bottom: 120px;
+}
+
+/* About Me */
+
+.about-me {
+  background: #1c1a19 url("resources/illustration-about-me-mobile.png") 100% 95% no-repeat;
+}
+
+.about-me .home-content {
+  width: 87%;
+  margin: auto;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto auto auto;
+  gap: 24px;
+  color: #fff;
+  padding: 120px 0 200px 0;
+}
```

Comment:
- [ ] I suggest that the content in the `about` section is aligned with `flexbox`. This will help us to make the about section easily responsive ☺️ .